docs/modules/player-lifecycle/basic-player-creation.md

```diff
@@ -0,0 +1,94 @@
+
+To do this, you need to:
+
+1. Set up your worker connector.
+1. Define the entity template for your player entities.
+1. Configure what entity template to use for player creation.
+
+## Set up your worker connector
+
+After setting up the module, you must enable it by adding the necessary systems to your workers. This is done in the `HandleWorkerConnectionEstablished()` method of your [`WorkerConnector`]({{urlRoot}}/reference/workflows/monobehaviour/creating-workers), by using the following code snippets:
+
+1. On a [client-worker]({{urlRoot}}/reference/glossary#client-worker): `PlayerLifecycleHelper.AddClientSystems(Worker.World)`
+1. On a [server-worker]({{urlRoot}}/reference/glossary#server-worker): `PlayerLifecycleHelper.AddServerSystems(Worker.World)`
+
+To change this behaviour, read the documentation on [custom player creation]({{urlRoot}}/modules/player-lifecycle/custom-player-creation).
+
+## Define the player entity template
+
+The module takes care of spawning the player entity as soon as a client-worker connects. To enable this behaviour, the server-worker responsible for handling requests to spawn player entities needs to know which [entity template]({{urlRoot}}/reference/concepts/entity-templates) to use when sending the entity creation request to the [SpatialOS Runtime]({{urlRoot}}/reference/glossary#spatialos-runtime).
+
+Create a method that returns an `EntityTemplate` object and takes the following parameters to define your player [entity template]({{urlRoot}}/reference/concepts/entity-templates):
+
+* `string workerId`: The ID of the worker that wants to spawn this player entity.
+*  `byte[] playerCreationArguments`: a serialized byte array of arguments provided by the worker instance sending the player creation request.
+
+When defining the entity template, you need to use the `AddPlayerLifecycleComponents` method. This method adds lifecycle SpatialOS components to the player entity template; these components are necessary to manage the player lifecycle for this entity.
+
+The following code snippet shows an example on how to implement such a method:
+
+```csharp
+public static class PlayerTemplate
+{
+    public static EntityTemplate CreatePlayerEntityTemplate(string workerId, byte[] playerCreationArguments)
+    {
+        // Obtain unique client attribute of the client-worker that requested the player entity
+        var clientAttribute = EntityTemplate.GetWorkerAccessAttribute(workerId);
+        // Obtain the attribute of your server-worker
+        var serverAttribute = "UnityGameLogic";
+
+        var entityTemplate = new EntityTemplate();
+        entityTemplate.AddPosition(new Position.Snapshot(new Coordinates()), serverAttribute);
+        // add all components that you want the player entity to have
+        AddPlayerLifecycleComponents(entityTemplate, workerId, serverAttribute);
+
+        return entityTemplate;
+    }
+}
+```
+
+## Configure an entity template to use for player creation
```

Comment:
This sounds suspiciously similar to the heading above

Configure the entity template delegate?